### DISCUSSION TITLE
Scenic II ECU Reset Help me !!

### DISCUSSION THREAD

**Discussion by Dawid198881** [cedricp/ddt4all#1152] [Q&A]
I reset my UCH without knowning why I did it. The cards don't work. I can't turn on the car. Can you help me??

**Comment by Dawid198881** [comment]
[ecu_ddt.txt](https://github.com/user-attachments/files/16045844/ecu_ddt.txt)
[screens.txt](https://github.com/user-attachments/files/16045838/screens.txt)

**Comment by cedricp** [comment]
There's a big warning in the virgin winodow that tells you to run away if you don't know what it is.
You'll need to pair the BCM with your key (card), here's how : 
* Go to "APPRENTISSAGE BADGE" then "APPRENTISSAGE APRES VENTE  2.0" screen
* Insert a card (key), then click on "APPRENDRE BADGE" in the lower right
* At this point the BCM has its ISK code recovered
* If you're lucky enough, your car is not an MK 2, so you can use the learn card plugin to extact PIN code and pair your cards with BCM. if it's an MK 2 you'll need a soft such as Renolink to do it.
Good luck

**Reply by Dawid198881** [reply]
![image](https://github.com/cedricp/ddt4all/assets/174268279/c93b2885-404c-4046-b7f2-c1867eae465b)
It doesn't work

**Reply by cedricp** [reply]
Is the "VSC ModeAPV_AppUCH_Arme" parameter set to 1 in "Etat du mode" ?
Did you insert a card in the card reader ??
I did that once and it worked flawlessly. 
Take a screenshot of the whole screen, please

**Reply by Dawid198881** [reply]
I cant't change this parameter

**Comment by cedricp** [comment]
Normal, it's a read-only one, but what is the value?

**Reply by Dawid198881** [reply]
0

**Reply by cedricp** [reply]
So, maybe it has it's ISK already set, have you tried to learn cards ?
